RELEASE/scripts/autoscend/auto_powerlevel.ash

```diff
@@ -52,6 +52,26 @@ boolean LX_attemptPowerLevel()
 		set_property("auto_powerLevelAdvCount", 0);
 		return true;		//restart the main loop to give those quests a chance to run now that the softblock is released.
 	}
+
+	if(get_property("screechDelay").to_boolean())
+	{
+		auto_log_warning("Patriotic Eagle's screech banished something we need and we can't adventure anywhere else");
+		if(get_property("screechCombats").to_int() > 0)
+		{
+			if(LX_getDigitalKey() || LX_getStarKey())
+			{
+				//If we still need the Digital Key or Star Key burn turns to get them
+				return true;
+			}
+			else
+			{
+				//Nothing else to do but go here
+				autoAdv($location[Noob Cave]);
+			}
+		}
+		set_property("screechDelay", false);
```

Comment:
this set should be inside an 
if(screechCombats == 0)
{
adv in noob cave to banish constructs
set prop to false
}
return true;

Reply:
this is to ensure we only clear the prop when we have banished constructs. Perhaps could even add another check to confirm constructs are banished after going to noob cave and aborting if they aren't. Perhaps abort is too strong... haven't thought too much on that. Will let you decide

Reply:
Yeah, I see. I forgot the part about going to Noob Cave to screech the constructs, but that only really applies to the L7 quest where we only fight undead and this part only gets called if there's nothing else to do, which is why I moved it to the bottom